vocabularies/Common.json

```diff
@@ -1288,6 +1288,132 @@
       "@Common.Experimental": true,
       "@Core.IsURL": true,
       "@Core.Description": "Base URL for WebSocket connections"
+    },
+    "RecommendationsFunction": {
+      "$Kind": "Term",
+      "$Type": "Common.QualifiedName",
+      "$AppliesTo": ["EntityType"],
+      "@Common.Experimental": true,
+      "@Core.Description": "Qualified name of a function that computes AI-based recommendations for entities of the annotated type (see [this template](#Template_RecommendationsFunction) and [this alternative](#Template_RecommendationsFunction2))"
+    },
+    "Template_RecommendationsFunction": [
+      {
+        "$Kind": "Function",
+        "$IsBound": true,
+        "@Common.Experimental": true,
+        "@Core.Description": "Template for functions that compute AI-based recommendations for the bound entity and its related entities",
+        "@Core.LongDescription": "If the function is invoked repeatedly in a stateful session, it SHOULD NOT\nrecompute recommendations whose predictors have not changed since the previous invocation.\n\nThe function computes the recommendations asynchronously and returns a URL where its result can be retrieved.\n\nThe template function itself cannot be invoked.",
+        "$Parameter": [
+          { "$Name": "Entity", "$Type": "Edm.EntityType" },
+          {
+            "$Name": "Targets",
+            "$Collection": true,
+            "@Core.IsURL": true,
+            "@Core.Description": "URLs (relative to the request URL) that address entities, properties or action/function parameters for which recommendations shall be retrieved",
```

Comment:
> URLs (relative to the request URL)

sounds recursive to me: If the client requests `SalesOrder('A')/ns.RecommendationsFunction?Targets=["","Items(10)"]`, that's the "request URL", isn't it? I do understand that only `SalesOrder('A')` is meant by the term, it becomes clear by the description. I just wonder if there is a better name for it?

Reply:
It's not recursive:
```
> new URL("Items(10)",`http://server/service/SalesOrder('A')/ns.RecommendationsFunction?Targets=["","Items(10)"]`).href
"http://server/service/SalesOrder('A')/Items(10)"
```
The part after the `?` of the "relative-to-URL" is ignored.

Reply:
> It's not recursive:

I know it isn't. I just wondered whether there is a better name, but maybe there isn't. Let's keep it.